tools/topology/topology2/production/tplg-targets-hda-generic.cmake

```diff
@@ -63,6 +63,12 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-ace1-2ch.bin,\
 DMIC0_ENHANCED_CAPTURE=true,EFX_DMIC0_TDFB_PARAMS=line2_generic_pm10deg,\
 EFX_DMIC0_DRC_PARAMS=dmic_default"
 
+# HDA + HDMI + DMICs for PTL
+"sof-hda-generic\;sof-hda-generic-ace3-2ch\;PLATFORM=ptl,HDA_CONFIG=mix,NUM_DMICS=2,\
```

Comment:
ACE3 should use the existing ace1 topology file if a DMIC blob is needed.
We don't have separate set of topologies for LNL (ACE2) either, the DMIC blob is the same.

@singalsu, can you clarify this?